plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/controller/FeatureDevController.kt

```diff
@@ -268,7 +304,7 @@ class FeatureDevController(
             }
             AmazonqTelemetry.isAcceptedCodeChanges(
                 project = null,
-                amazonqNumberOfFilesAccepted = (filePaths.size + deletedFiles.size) * 1.0,
+                amazonqNumberOfFilesAccepted = (filePaths.filterNot { it.rejected }.size + deletedFiles.filterNot { it.rejected }.size) * 1.0,
```

Comment:
in vscode we calculate differently [link](https://github.com/aws/aws-toolkit-vscode/pull/4523/files#diff-3126666ce5709a5f19fcabd5c734242bf55ac7ba9dff730b5393d68911693012R408)

can you add a task to a board to fix vscode metrics please?

Reply:
Yes, done: https://taskei.amazon.dev/tasks/D127722149